### PR TITLE
Use the right package version when installing Librarian

### PR DESCRIPTION
**Pull request by josecelano**

### CODE REVIEW

**Comment by josecelano** [comment]
The first execution of the workflow has installed the 0.2.0 version:

https://github.com/Nautilus-Cyberneering/chinese-ideographs/runs/4474500128?check_suite_focus=true#step:7:142
image

It's ok becuase it was the current head for main branch.

**Comment by josecelano** [comment]
I've added a new commit to the main branch on the Librarian and now this is the result:

https://github.com/Nautilus-Cyberneering/chinese-ideographs/runs/4474577095?check_suite_focus=true#step:7:142
image
so It seems work. We should add a new command to the Librarian to show the version. Like @da2ce7 did on the [nautilus namecodes repo](https://github.com/da2ce7/nautilus-namecodes).

```
nautilus-librarian --version

File: .github/workflows/gold-drawings-processing.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
           python-version: '3.9'
 
       - name: Setup Poetry
-        run: pip3 install poetry
+        run: pip3 install poetry poetry-dynamic-versioning
```
[the remark this answers is not in the document]
Comment (josecelano) [reply .github/workflows/gold-drawings-processing.yml:26]: I think we need in order to get the right version with `poetry version --short`.Do you men we should add first the dependency and later the new feature?

[the remark this answers is not in the document]
Comment (da2ce7) [reply .github/workflows/gold-drawings-processing.yml:26]: The version command doesn't depend on `poetry-dynamic-versioning`: https://python-poetry.org/docs/cli/#version

The `poetry-dynamic-versioning` takes the version from the latest git tag instead from hard-coded in the `pyproject.toml` file.

[the remark this answers is not in the document]
Comment (josecelano) [reply .github/workflows/gold-drawings-processing.yml:26]: I think the `poetry-dynamic-versioning` also changes the behavior of the `poetry version` command (not `poetry --version`). For example if I run it with the main branch of the librarian I get:

```shell
$ poetry version
nautilus-librarian 0.2.1.post2.dev0+b6c2ed0
```

But if you change the `pyproject.toml` disabling the plugin:

```
[tool.poetry-dynamic-versioning]
enable = false
```

And you run again `poetry version` you will get:

```
$ poetry version
nautilus-librarian 0.0.0
```

That means my second change would have not worked without installing `poetry-dynamic-versioning`. But maybe I'm missing something.

[the remark this answers is not in the document]
Comment (josecelano) [reply .github/workflows/gold-drawings-processing.yml:26]: I'm going to merge this PR. I've created an issue on the [plugin repo](https://github.com/mtkennerly/poetry-dynamic-versioning/issues/67) just for adding a clarification on theirs docs.

**Pull request merged**